src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task05.java

```diff
@@ -31,8 +31,49 @@ public static void main(String[] args) {
      * @return месяц на английском языке (UPPER CASE)
      */
     static String getMonthName(byte monthNumber) {
+        String monthName = "";
+        switch (monthNumber){
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"